docs/user_guide/remote_processing.rst

```diff
@@ -7,98 +7,51 @@ The SiliconCompiler project supports a remote processing model that leverages th
  #. Warehouse scale elastic compute.
  #. NDA encumbered IPs, PDKs, and EDA tools.
 
-For command line compilation, remote processing is turned on with the '-remote' option. Results from a remote compilation should be identical to results from a local compilation.
+Note that our public beta currently only supports open-source tools and PDKs. You can access the public beta without a signup or login, and it is designed to delete your data after your jobs finish, but it is not intended to process proprietary or restricted intellectual property! Please `review our terms of service <https://www.siliconcompiler.com/terms-of-service>`_, and do not submit IP which you are not allowed to distribute.
+
+Even though our publicly-available servers only support open-source IP and tools, the remote API is capable of supporting any SiliconCompiler modules which the server operators wish to install, and we provide a minimal example development server which can be used as a starting point for custom server implementations. You can also find descriptions of the core remote API calls in the :ref:`remote API <Server API>` section.
+
+Quickstart Guide
+----------------
+
+To perform a minimal self-test using the remote flow, you can run the following commands::
+
+    pip install siliconcompiler
+    sc -target asic_demo -remote
+
+The job should only take a few minutes to run if the servers aren't too busy, and once it completes, you should receive a PDF file containing a screenshot and metrics for the build results. The self-test design is a simple 8-bit counter, so your results should look like a small block of digital logic.
+
```

Comment:
Add the screenshot here. That way you don't really need the "so your results should look like a small block of digital logic." part